### DISCUSSION TITLE
LDAP support - can users login with LDAP credentials?

### DISCUSSION THREAD

**Discussion by Galileo80** [pivpn/pivpn#1872] [Q&A]
Hi, the PiVPN project looks very good! I couldn't find if PiVPN also supports the LDAP protocol and if you can authenticate the VPN users against an external or local LDAP server?

**Comment by orazioedoardo** [comment]
It’s not built in in WireGuard nor OpenVPN but there are likely plugins that can offer LDAP on top of them, especially with OpenVPN.

**Comment by Galileo80** [comment]
Ok cool, does that require changinf the conf files that client certificates are not necesarry? Because the server won’t generate them.

Would be nice if this was added to the package like it is on Synology DSM. 👍

**Reply by orazioedoardo** [reply]
> Ok cool, does that require changinf the conf files that client certificates are not necesarry? 

Probably it works with username and password as opposed to client certificate
